app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MenusController < ApplicationController
+  def show
+    @menus = Menu.where("restaurant_id = ?", params[:restaurant_id].to_i)
```

Comment:
Found another way. Not sure why it did not work yesterday (as we use the same method for restaurant categories and restaurants). Hopefully more "railsy" :)
The method above was from the ActiveRecord documentation (section 2.2) though.